### ISSUE TITLE
[question] cookiecutter template

### ISSUE THREAD

**Issue by vsoch** [binder-examples/continuous-build#10]
to make generation one step easier, I'm starting to test using [cookiecutter](https://github.com/audreyr/cookiecutter), and have an example that (doesn't do much) other than to spit out the repo with a few variables defined. Would this be of interest for the binder-examples tooling? --> https://github.com/vsoch/cookie-builder I'm going to write up a full tutorial to go from nothing --> deployed notebook on a slurm cluster (in progress here https://vsoch.github.io/lessons/sherlock-singularity/) as soon as we get #9 fixed up for full functionality.

**Comment by jhamman** [comment]
I have just developed a custom binder-cookiecutter template for the [Pangeo Project](pangeo.io): https://github.com/pangeo-data/cookiecutter-pangeo-binder

In talking with @choldgraf - this may be useful to expand/generalize to other binder applications.

**Comment by choldgraf** [comment]
agreed - perhaps we could move this issue over to the `repo2docker` or `binder` repositories? These binder-examples repos get less attention :-)
